plugins/CoreConsole/Commands/DevelopmentSyncProcessedSystemTests.php

```diff
@@ -34,12 +34,19 @@ protected function configure()
         $this->setDescription('For Piwik core devs. Copies processed system tests from travis artifacts to local processed directories');
         $this->addArgument('buildnumber', InputArgument::REQUIRED, 'Travis build number you want to sync, eg "14820".');
         $this->addOption('expected', 'e', InputOption::VALUE_NONE, 'If given file will be copied in expected directories instead of processed');
+        $this->addOption('repository', 'r', InputOption::VALUE_OPTIONAL, 'Repository name you want to sync screenshots for.', 'matomo-org/matomo');
+        $this->addOption('http-user', '', InputOption::VALUE_OPTIONAL, 'the HTTP AUTH username (for premium plugins where artifacts are protected)');
+        $this->addOption('http-password', '', InputOption::VALUE_OPTIONAL, 'the HTTP AUTH password (for premium plugins where artifacts are protected)');
+
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->updateCoreFiles($input, $output);
-        $this->updatePluginsFiles($input, $output);
+
+        if (!$input->getOption('repository')) {
```

Comment:
won't that always return a value? Guess we would need to compare it's `matomo-org/matomo`

Reply:
👍 updated it